src/gset.rs

```diff
@@ -1,12 +1,15 @@
 use std::collections::BTreeSet;
 
+use serde::{de, ser};
+use serde_derive::{Serialize, Deserialize};
+
 /// A `GSet` is a grow-only set.
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
-pub struct GSet<A: Ord + Serialize + DeserializeOwned> {
+pub struct GSet<A: Ord> {
     value: BTreeSet<A>,
 }
 
-impl<A: Ord + Serialize + DeserializeOwned> GSet<A> {
+impl<A: Ord + ser::Serialize + de::DeserializeOwned> GSet<A> {
```

Comment:
We shouldn't need `Serialize` and `DeserializeOwned` bounds on this `A`, lets remove it.
Also, mind renaming the type variable `A` to `T` or something? We usually use `A` for the `Actor` type variable in this library.

Reply:
Ack, done. Clippy also suggested deriving `Default` on it.

Reply:
Yeah `Default` is nice to have